host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1265,4 +1266,12 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @LogMessage(level = Level.ERROR)
     @Message( id = 172, value = "Failed executing the suspend operation for server: %s")
     void suspendExecutionFailed(@Cause IOException cause, String serverName);
+
+    @LogMessage(level = Level.INFO)
```

Comment:
Thanks. The github UI is hiding this discussion because the original line I commented on has been changed. :)